### ISSUE TITLE
Unable to build iOS app, missing font?

### ISSUE THREAD

**Issue by wittjosiah** [mauron85/react-native-background-geolocation-example#5]
Not sure if I missed a step when trying to setup, but I've been unable to get the iOS app to build for the simulator. Have you seen this issue before, any thoughts on what's going wrong here?

## Environment
- macOS 10.13.4
- iOS 11.3 Simulator

## Expected Behaviour
- App builds

## Actual Behaviour
- App does not build

## Steps to Reproduce
- Clone repo
- Run `npm install`
- Run `react-native link react-native-mauron85-background-geolocation`
- Run `react-native run-ios`

## Debug Logs

```
...
CpResource /Users/josiah/projects/react-native-background-geolocation-example/node_modules/native-base/Fonts/rubicon-icon-font.ttf build/Build/Products/Debug-iphonesimulator/RNBGExample.app/rubicon-icon-font.ttf
    cd /Users/josiah/projects/react-native-background-geolocation-example/ios
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/Users/josiah/Library/Python/3.6/bin:/usr/local/bin:/usr/local/sbin:/Users/josiah/.nvm/versions/node/v8.11.1/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Applications/Keybase.app/Contents/SharedSupport/bin:/Users/josiah/Library/Android/sdk/tools:/Users/josiah/Library/Android/sdk/platform-tools"
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks /Users/josiah/projects/react-native-background-geolocation-example/node_modules/native-base/Fonts/rubicon-icon-font.ttf /Users/josiah/projects/react-native-background-geolocation-example/ios/build/Build/Products/Debug-iphonesimulator/RNBGExample.app
error: /Users/josiah/projects/react-native-background-geolocation-example/node_modules/native-base/Fonts/rubicon-icon-font.ttf: No such file or directory


** BUILD FAILED **


The following build commands failed:
	CpResource /Users/josiah/projects/react-native-background-geolocation-example/node_modules/native-base/Fonts/rubicon-icon-font.ttf build/Build/Products/Debug-iphonesimulator/RNBGExample.app/rubicon-icon-font.ttf
(1 failure)

Installing build/Build/Products/Debug-iphonesimulator/RNBGExample.app
An error was encountered processing the command (domain=NSPOSIXErrorDomain, code=22):
Failed to install the requested application
The bundle identifier of the application could not be determined.
Ensure that the application's Info.plist contains a value for CFBundleIdentifier.
Print: Entry, ":CFBundleIdentifier", Does Not Exist

Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier build/Build/Products/Debug-iphonesimulator/RNBGExample.app/Info.plist
Print: Entry, ":CFBundleIdentifier", Does Not Exist
```

**Comment by mauron85** [comment]
Unfortunately [nativebase](https://nativebase.io/) module doesn't follow [semantic versioning](https://semver.org/) and it seems they removed some fonts in their minor update. You can fix that, by explicitly installing version 2.3.1

```
npm i native-base@2.3.1
```

**Comment by wittjosiah** [comment]
Yup that worked! Thanks @mauron85!
